packages/compass-web/src/entrypoint.tsx

```diff
@@ -277,15 +279,22 @@ const CompassWeb = ({
   const autoConnectConnectionId = initialWorkspaceRef.current?.connectionId;
 
   const telemetry = useRef({
-    createTrack:
-      () =>
-      async (event: string, properties: Record<string, any> | undefined) => {
-        onTrack &&
-          void onTrack(
-            event,
-            typeof properties === 'function' ? await properties() : properties
-          );
-      },
+    createTrack: (logger: Logger, preferences: TelemetryPreferences) =>
+      createGenericTrack({
+        sendTrack: async (
+          event: string,
+          properties: Record<string, any> | undefined
+        ) => {
+          onTrack &&
+            void onTrack(
+              event,
+              typeof properties === 'function' ? await properties() : properties
```

Comment:
Properties are never a function

```suggestion
              properties
```